src/app/friends/view-friends-list/view-friends-list.component.ts

```diff
@@ -7,4 +9,29 @@ import { Component } from '@angular/core';
 })
 export class ViewFriendsListComponent {
 
+
+  friends = ['Ram', 'Harry', 'Raj'];
+
+  currentUrl !: string;
+  isFriendActive : boolean = false;
+  friendName: any;
+  constructor(public router : Router, public route: ActivatedRoute){
+    this.route.params.subscribe(params => {
```

Comment:
Add comment to inform what this.route.params.subscribe does

Reply:
Added comments.